Use updated weasyprint due to changes in pypdf class usage

BFH-hga3 opened the pull request:
This fixed #6 . Please review!

BFH-hga3 committed:
Use updated weasyprint due to cheanges in pypdf class usage

iSOLveIT commented on pyproject.toml:
```diff
@@ -39,7 +39,7 @@ sphinx = ">=5.0"
 docutils = ">=0.15"
 pathlib = ">=1.0"
-weasyprint = ">=60.0, <61.0"
+weasyprint = ">=62.0"
```
Please change the Python version to 3.10 and above since `weasyprint >=62.0` requires `Python >=3.9`
``` toml
python = ">=3.10.0,<4.0"
```
After the change, you can update the `poetry.lock` using `poetry update`.

BFH-hga3 replied:
THX! I updated as you suggested.

BFH-hga3 committed:
Fix minimal python version due to weasyprint requirements

THX Randy Duodu for pointing this out. See https://doc.courtbouillon.org/weasyprint/stable/changelog.html#version-62-0 for further information

BFH-hga3 committed:
Updated due to weasyprint python version requirements

iSOLveIT reviewed (approved):
Thank you for your contribution to the plugin. 🫡
